api/HTMLMediaElement.json

```
@@ -3569,9 +3569,17 @@
             },
             "chrome_android": "mirror",
             "edge": "mirror",
-            "firefox": {
-              "version_added": "52"
-            },
+            "firefox": [
+              {
+                "version_added": "133"
+              },
+              {
+                "version_added": "52",
+                "version_removed": "133",
+                "partial_implementation": true,
+                "notes": "`onwaitingforkey` content attribute not implemented."
+              }
```

Do you know why we keep this attribute? What works in 52-132, and what doesn't work?

---

@caugner So in 52-133 you can add a JavaScript handler for the `waitingforkey` event using `addEventListener`, but you can't use the `onwaitingforkey` attribute in HTML to add an inline handler for the event. Hence a partial implementation.
In 133 the  `onwaitingforkey` attribute in HTML was added.

I did it this way because this seemed like a partial implementation. But we could pretend it was always a full implementation and just add a note to the other feature `"notes": "version 133 adds support for `onwaitingforkey` content attribute."`

---

No, there should never be an `on*` feature. [The relevant data guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#dom-events-eventname_event) says to:

> use `partial_implementation` with the note <code>"The \`onfocus\` event handler property is not supported."</code>. If only the `on` event handler property is supported and not the event itself, use `"version_added": false`.

```suggestion
              {
                "version_added": "52",
                "version_removed": "133",
                "partial_implementation": true,
                "notes": "The `onwaitingforkey` event handler property is not supported."
              }
```

---

I've merged this - good to know there is standard text.